WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -50,10 +50,29 @@ final class ProductFormViewController: UIViewController {
 private extension ProductFormViewController {
     func configureNavigationBar() {
         let updateTitle = NSLocalizedString("Update", comment: "Action for updating a Product remotely")
-        navigationItem.rightBarButtonItem = UIBarButtonItem(title: updateTitle, style: .done, target: self, action: #selector(updateProduct))
+        navigationItem.rightBarButtonItems = [UIBarButtonItem(title: updateTitle, style: .done, target: self, action: #selector(updateProduct))]
+
+        if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.editProductsRelease2) {
+            navigationItem.rightBarButtonItems?.insert(moreOptionsBarButton(), at: 0)
+        }
         removeNavigationBackBarButtonText()
     }
 
+    func moreOptionsBarButton() -> UIBarButtonItem {
+        let moreButton = UIBarButtonItem(image: .moreImage,
+                                     style: .plain,
+                                     target: self,
+                                     action: #selector(presentMoreOptionsActionSheet))
+        moreButton.accessibilityTraits = .button
+        moreButton.accessibilityLabel = NSLocalizedString("Show more options", comment: "Accessibility label for the Edit Product More Options action sheet")
+        moreButton.accessibilityHint = NSLocalizedString(
+            "Action for show the More Options action sheet in Edit Product.",
+            comment: "VoiceOver accessibility hint, informing the user the button can be used to show the More Options action sheet."
+        )
+        moreButton.accessibilityIdentifier = "edit-product-more-options-button"
+        return moreButton
```

Comment:
First, thank you for thinking about accessibility! 🙏 🙇 `UIBarButtonItem` is very easy to miss because they're usually just icons. 

Some suggestions:

1. The `acccessibilityTraits` is not needed. The fact that it's a `UIBarButtonItem` will give VoiceOver enough information and will treat it automatically as a `.button`. 
2. The `accessibilityLabel` can just be "More" or "More options". That's understandable enough. Labels should be brief. 🙂 
3. The `accessibilityHint` should start with a verb. There are some guidelines [here](https://developer.apple.com/library/archive/documentation/UserExperience/Conceptual/iPhoneAccessibility/Making_Application_Accessible/Making_Application_Accessible.html#//apple_ref/doc/uid/TP40008785-CH102-SW11) for how to improve it. But I suggest we just **exclude** it. The accessibility label already describes the button well IMO. 

    > You should provide a hint only when the results of an action are not obvious from the element’s label.

Reply:
Thanks @shiki for these useful suggestions 😄 TIL new things on accessibility. We should update also other points of the app where we use the same approach.